.github/workflows/ci-model-regression.yml

```diff
@@ -87,6 +88,7 @@ jobs:
 
     - name: Update the comment with the configuration
       uses: peter-evans/create-or-update-comment@v1
+      if: steps.fc_config.outputs.comment-id != ''
```

Comment:
How about using https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error instead of all the the `if`s?

Reply:
Previously the workflow fails if a configuration couldn't be found, and it was ok, but a user saw that the workflow failed and it could be misleading for him. It's the main reason why I used `if`s, it helps to have green workflow and red only if something really bad happened 😄